assets/stylesheets/bootstrap/_input-groups.scss

```diff
@@ -21,7 +21,12 @@
     // proper border colors.
     position: relative;
     z-index: 2;
+
+    // IE9 fubars the placeholder attribute in text inputs and the arrows on
+    // select elements in input groups. To fix it, we float the input. Details:
+    // https://github.com/twbs/bootstrap/issues/11561#issuecomment-28936855
```

Comment:
we don't care about IE9.  We'll keep the float since it doesn't harm anything and only remove it if needed.